### PR TITLE
Update report.py

### PR DESCRIPTION
**Pull request by pythexcel**

### CODE REVIEW

File: app/api/report.py [hunk 1]
```diff
@@ -1057,3 +1057,126 @@ def junior_monthly_report():
 
     return jsonify(report_all)
 
+
+#Api for get juniors managers details.
+@bp.route('/get_managers', methods=['GET'])
```
Comment (reactexcel) [inline app/api/report.py]: move all 360 route to a file called 360.py
same for montly. all functions/routes for montly to montly.py

File: app/api/report.py [hunk 2]
```diff
@@ -1057,3 +1057,126 @@ def junior_monthly_report():
...
+        return jsonify(
+            {"msg": "currently you don't have any manager assigned"}),204
+    else:
+        return jsonify(managers)
+
+
+#api for 360_review
+@bp.route("/360_reviews", methods=["GET", "POST"])
+@jwt_required
+def reviews_360():
```
Comment (reactexcel) [inline app/api/report.py]: where is this used?

File: app/api/report.py [hunk 3]
```diff
@@ -1057,3 +1057,126 @@ def junior_monthly_report():
...
+def get_reviews():
+    reviewss = [ ]
+    review = mongo.db.reviews_360.find({
+        "anon": False
+        })
+    review = [serialize_doc(doc) for doc in review]
+    for doc in review:
+        reviewss.append(doc)
+
+    docss = mongo.db.reviews_360.find({
```
Comment (reactexcel) [inline app/api/report.py]: why do we have two quries with anan true/false.

File: app/api/report.py (hunk 2 above)
Comment (pythexcel) [reply app/api/report.py]: This route is using for sending juniors comments to managers using POST methods . and get methods for user can see his response.

File: app/api/report.py (hunk 3 above)
Comment (pythexcel) [reply app/api/report.py]: anon is true for anonymous users and anon is false for normal users.

File: app/api/threesixty.py [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+from flask_jwt_extended import (jwt_required, get_current_user
+)
+
+
+bp = Blueprint('threesixty', __name__, url_prefix='/')
+
+#Api for get juniors managers.
+@bp.route('/get_managers', methods=['GET'])
+@jwt_required
+def get_managers():
```
Comment (reactexcel) [inline app/api/threesixty.py]: hmm. does this api already exist. this look like something which would be used in dashboard as well??

this code should be reused as this is used in multiple places

File: app/api/report.py (hunk 3 above)
Comment (reactexcel) [reply app/api/report.py]: yes i get that but why 2. why not just one

File: app/api/threesixty.py (hunk 4 above)
Comment (pythexcel) [reply app/api/threesixty.py]: No this is only single API for getting juniors managers. in the dashboard, frontend is showing all details from profile. and all other api is to find managers juniors.

**Pull request merged**